.github/workflows/example.yml

```diff
@@ -11,6 +11,33 @@ jobs:
   checkoutAndBuild:
     runs-on: ubuntu-latest
     steps:
+
+      # Fetch name of the Forked Repository with Branch
+      # if workflow is triggered from pull request
+      - name: Fetch PR head repo and branch name
+        if: github.event_name == 'pull_request'
+        run: |
+          HEAD_REPO_NAME=$(jq -r '.pull_request.head.repo.full_name' "$GITHUB_EVENT_PATH")
+          echo "PR head repo: $HEAD_REPO_NAME"
```

Comment:
Can you make the env vars uppercase?

Reply:
In the GH docs https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable they have mentioned that in lowercase, so I also did the same way.

Reply:
May the way you suggested is the better way.

Reply:
Environment variables should be uppercase, variables within scripts are often lowercase. I don't agree with the GH docs on that point.

Reply:
Yes, it makes sense. I also did the same way but changed after seeing GH docs :smile: